sdk/schemaregistry/azure-data-schemaregistry-jsonschema/src/samples/java/com/azure/data/schemaregistry/jsonschema/ReadmeSamples.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.schemaregistry.jsonschema;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.http.ContentType;
+import com.azure.core.models.MessageContent;
+import com.azure.core.util.BinaryData;
+import com.azure.core.util.serializer.TypeReference;
+import com.azure.data.schemaregistry.SchemaRegistryAsyncClient;
+import com.azure.data.schemaregistry.SchemaRegistryClientBuilder;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.messaging.eventhubs.EventData;
+
+public class ReadmeSamples {
+
+    /**
+     * Sample to demonstrate creation of {@link SchemaRegistryJsonSchemaSerializer}.
+     *
+     * @return The {@link SchemaRegistryJsonSchemaSerializer}.
+     */
+    public SchemaRegistryJsonSchemaSerializer createSerializer() {
+        // BEGIN: readme-sample-createSchemaRegistryAsyncClient
+        TokenCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
+
+        // {schema-registry-endpoint} is the fully qualified namespace of the Event Hubs instance. It is usually
+        // of the form "{your-namespace}.servicebus.windows.net"
+        SchemaRegistryAsyncClient schemaRegistryAsyncClient = new SchemaRegistryClientBuilder()
+            .fullyQualifiedNamespace("{your-event-hubs-namespace}.servicebus.windows.net")
+            .credential(tokenCredential)
+            .buildAsyncClient();
+        // END: readme-sample-createSchemaRegistryAsyncClient
+
+        // BEGIN: readme-sample-createSchemaRegistryJsonSchemaSerializer
+        SchemaRegistryJsonSchemaSerializer serializer = new SchemaRegistryJsonSchemaSerializerBuilder()
+            .schemaRegistryClient(schemaRegistryAsyncClient)
+            .schemaGroup("{schema-group}")
+            .buildSerializer();
```

Comment:
Will this throw a NPE as this is not setting the `jsonSchemaGenerator` and the build method validates that this is not null?

Reply:
Good catch!